inc/sitemaps/class-sitemap-image-parser.php

```diff
@@ -67,7 +67,14 @@ public function get_images( $post ) {
 			$images[] = $this->get_image_item( $post, $src, $title, $alt );
 		}
 
-		$unfiltered_images = $this->parse_html_images( $post->post_content );
+		/**
+		 * Filters the post content before it is parsed for images.
+		 *
+		 * @param string $content
```

Comment:
Can you add some docs about the $content` 

```
@param string $content The post content.
```

It might be better to use this format for documenting the hook:

			/**
			 * Filter: 'wpseo_sitemap_content_before_parse_html' - Filters the post content before it is parsed for images.
			 *
			 * @api string $content The post content.
			 */